apps/twitch-bot/internal/command/invite.go

```diff
@@ -6,29 +6,27 @@ import (
 	"strings"
 
 	"github.com/gempir/go-twitch-irc/v3"
-	"github.com/senchabot-dev/monorepo/apps/twitch-bot/client"
 	"github.com/senchabot-dev/monorepo/apps/twitch-bot/internal/models"
-	"github.com/senchabot-dev/monorepo/apps/twitch-bot/server"
 )
 
-func InviteCommand(client *client.Clients, server *server.SenchabotAPIServer, message twitch.PrivateMessage, commandName string, params []string) {
+func (s *commands) InviteCommand(message twitch.PrivateMessage, commandName string, params []string) {
 	if message.Channel != "senchabot" {
 		return
 	}
 
 	if len(params) < 1 {
-		client.Twitch.Say(message.Channel, "!invite [your_channel_name]")
+		s.client.Twitch.Say(message.Channel, "!invite [your_channel_name]")
 		return
 	}
 
 	var channelName = message.User.Name
 	if strings.ToLower(params[0]) != channelName {
-		client.Twitch.Say(message.Channel, "You need to specify your channel name. !invite "+channelName)
+		s.client.Twitch.Say(message.Channel, "You need to specify your channel name. !invite "+channelName)
 		return
 	}
 
 	var twitchChannelId = message.User.ID
-	alreadyJoined, err := server.CreateTwitchChannel(context.Background(), twitchChannelId, channelName)
+	alreadyJoined, err := s.service.DB.CreateTwitchChannel(context.Background(), twitchChannelId, channelName, nil)
```

Comment:
[for follow up pr]

let's create the context at handler level so that we can make sure everything is using the same context when a request happens.